Automattic/wp-calypso
# Plans 2023: Always show the biannual toggle. free/paid site plans

Fixes https://github.com/Automattic/wp-calypso/issues/74636
Based on https://github.com/Automattic/wp-calypso/pull/74722

## Proposed Changes

This will show the 1yr/2yr toggle on both paid/free site plans (irrespective of billing term) when visiting `/plans/[SITE]` with the `plans/biannual-toggle` flag enabled. 

Previously, the toggle was only visible on free sites and on `/plans/monthly/[SITE]` with site on a monthly plan due to the "monthly" option and monthly restrictions (downgrading from yearly to monthly not being permissible).

### Media

<img width="700" alt="Screenshot 2023-03-22 at 4 04 28 PM" src="https://user-images.githubusercontent.com/1705499/226931799-fda7f70f-57c3-4403-9310-62c70928b280.png">


## Testing Instructions



* Go to `plans/[SITE]` with both free/paid sites and feature flag enabled `?flags=plans/biannual-toggle`
* The 1yr/2yr toggle should always be visible/active

## Pre-merge Checklist
